### DISCUSSION TITLE
Advanced Installer?

### DISCUSSION THREAD

**Discussion by rollingmoai** [nefarius/ViGEmBus#136] [Support]
Not something I'd expect. An open-source project switching from WiX to a proprietary solution? It's not like what Advanced Installer can do WiX can't do??

**Comment by nefarius** [comment]
Auto-updater, signing helper, less overhead,... After all I have to deal with the packaging and publishing overhead, anybody else is welcome to waste their time, I am not and I don't see how that decision has anything to do with the entire project remaining FOSS... It's easy to complain if one doesn't contribute... But let me help you burst the bubble:

- I am and never was under the slightest obligation to even **provide** any kind of setup, I did it as a community service to make the project "mainstream", yes it is FOSS but that doesn't mean I **owe** anybody any setup (the entitlement...)
- What difference does it make for 99.9% of the users what setup engine I use? That's right, none whatsoever. No good argument has come forth other than fake "outrage". "Not something I'd expect." isn't valid criticism.
- The top priorities for a setup are **not** that it satisfies the GNU bros, but
  - That it works rock solid on thousands, if not millions on machines with different configurations
  - That I, the author, have little to no overhead with it (yes, setups don't fall from trees, they take a lot of time).
  - That thanks to using a licensed product I am entitled to Enterprise support, should I ever need it. If I have problems with WiX I can burn my own weekends trying to troubleshoot but that's fine since it's not **your** time I guess
- You have to trust me nonetheless, FOSS drivers give you the illusion of control and security, who can guarantee that the binaries and setups remain unaltered after I have to submit them to Microsoft for signing? Nobody, we have to trust the entire supply chain. That's the simple reality.

Last but not least who are the one's "profiting" from a FOSS setup? Those who bundle the software and don't credit properly and that time has ended now too. If you wanna have the cake and also eat it, you can invest in an Advanced Installer Enterprise license. Or not and just keep using my builds and accept the way I operate. Ez gg.
